automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -109,20 +121,31 @@ object Leonardo extends RestClient with LazyLogging {
 
     def listIncludingDeleted()(implicit token: AuthToken): Seq[Cluster] = {
       val path = "api/clusters?includeDeleted=true"
-      // logger.info(s"Listing all clusters including deleted: GET /$path")
```

Comment:
Should we preserve this log message? We typically log in this class when a Leo API is called.

Reply:
Makes sense. It had been commented out and I assumed it wasn't necessary.